src/app/(nextjs_migration)/(guest)/breach-details/[breachName]/page.tsx

```diff
@@ -0,0 +1,368 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Image, { StaticImageData } from "next/image";
```

Comment:
Yeah, we agreed to switch to Prettier for formatting (it's more complete, and doesn't show up as warnings before it's been formatting), since we're creating many new files anyway. So TS files are formatted by Prettier, while existing files are left as-is.

When the migration is complete, we might still want to do a mega-formatting commit for the remaining files.

Reply:
Filed https://github.com/mozilla/blurts-server/issues/3055